scipy/stats/stats.py

```diff
@@ -1037,8 +1040,32 @@ def moment(a, moment=1, axis=0):
        is None. The denominator for the moment calculation is the number of
        observations, no degrees of freedom correction is done.
 
+    See also
+    --------
+    kurtosis, skew, describe
+
+    Notes
+    -----
+    [1]_ The n-th moment of a real-valued continuous function f(x) of a real
+    variable about a value c is:
+        \mu _{n} = \int_{-\infty }^{\infty }(x - c)^n  f(x) dx
```

Comment:
Since you asked to review the the TeX notation: 

* it needs to have blanks line above and below.
* it needs to have the reST magic to render: `.. math::`  (notice the space between after the dots)

I personally prefer to minimize the number of curly brackets (here: `\infty` is a single symbol for TeX, hence `_{\infty}` and `_\infty` are the same for TeX). Consistent spacing helps readability, too. Here's how I'd write it:

```

.. math::

    \mu_n  = \int_{-\infty}^\infty (x - c)^n f(x) dx
```

(When typesetting, you'd add `\,` before dx, but it clutters the plain text version, so don't).

In any case, this formula should not be here --- better add a formula for what is actually being calculated.